locales-src/newtab.ftl

```diff
@@ -36,3 +36,15 @@ newtab-topsites-url-validation = Valid URL required
 newtab-topsites-image-url-label = Custom Image URL
 newtab-topsites-use-image-link = Use a custom image…
 newtab-topsites-image-validation = Image failed to load. Try a different URL.
+
+newtab-topsites-cancel-button = Cancel
+newtab-topsites-delete-history-button = Delete from History
+newtab-topsites-save-button = Save
+newtab-topsites-preview-button = Preview
+newtab-topsites-add-button = Add
+
+newtab-confirm-delete-history-p1 = Are you sure you want to delete every instance of this page from your history?
+# p1 and p2 are in the same delete dialog. "This action" refers to deleting a page from history.
+newtab-confirm-delete-history-p2 = This action cannot be undone.
```

Comment:
Fluent allows `<br/>`s. @flodolo should we just make this one string "…from your history?<br/><br/>This action…" ?

Doesn't look like the text is styled differently for the first vs second `<p>`:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/438537/59133535-3a877400-892d-11e9-9fea-b20e88517b01.png">

Reply:
Although it does seem a bit odd to expose the double `<br/>`…

Reply:
I'd keep them separate (as they currently are).